tests/fcgi-responder.c

```diff
@@ -44,6 +44,10 @@ int main (void) {
 			printf("%s", getenv("PATH_INFO"));
 		} else if (0 == strcmp(p, "script_name")) {
 			printf("%s", getenv("SCRIPT_NAME"));
+		} else if (0 == strcmp(p, "var")) {
+			char *test = getenv("LIGHTYTEST");
+			if (test == NULL) test = "(no value)";
+			printf("%s", test);
```

Comment:
I don't see why the Content-Type is needed, or the subsequent body since it should be discarded for 200 OK (but should be preserved for the 403 Forbidden).  I added it back when I noticed that a response body was being sent unconditionally.

I don't have a strong preference for or against its presence, as long as the tests pass.